src/util/definitions.jl

```diff
@@ -1,9 +1,19 @@
-# DEFAULT outage data which is used when outage_flag is set to FALSE
-# Based on ERCOT historical data
+# TODO: OUTAGE_INFO FILE: this should probably be an artifact 
+"""
+    DEFAULT outage data which is used when outage_flag is set to FALSE
+
+Based on ERCOT historical data
+"""
 const OUTAGE_INFO_FILE =
     joinpath(@__DIR__, "descriptors", "outage-rates-ERCOT-modified.csv")
 
-# ACBranch Types
+"""
+    Supported DC branch Types
+"""
 const HVDCLineTypes = Union{PSY.TwoTerminalHVDCLine, PSY.TwoTerminalVSCDCLine}
+
+"""
+    Supported Transformer Types
```

Comment:
These are not really supported XF types. PRAS doesn't really care about XF's.

Reply:
How about filtered transformer types?